### PR TITLE
Implement proposed API

### PR DESCRIPTION
**Pull request by jessemyers**
See README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,15 +42,15 @@ With this `Metadata`, `Nodule` can load configuration using loader functions:
         import { Nodule, loadFromEnvironment } from 'nodule-config';
 
         const nodule = new Nodule(metadata);
-        nodule.loader(loadFromEnvironment).then(bottle => myInitFunc(bottle));
+        nodule.from(loadFromEnvironment).load().then(bottle => myInitFunc(bottle));
```
Comment (jessemyers) [inline README.md]: I changed the API slightly:
 - I need a "build" function on my builder to initiative the async part
 - I chose to use `from` to differentiate loader configuration from the act of loading

File: src/bind.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import getInjector from './injector';
```
Comment (jessemyers) [inline src/bind.js]: No more decorators

File: src/injector.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Bottle } from 'bottlejs';
+
+
+export default function getInjector(scope = null) {
```
Comment (jessemyers) [inline src/injector.js]: I read the Bottle docs and they have their own global state. We don't need our own...

Comment (pelegdvir) [reply src/injector.js]: and multiple scopes too

File: src/nodule.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    loadFromCredstash,
+    loadFromEnvironment,
+    loadFromObject,
+} from './loaders';
+import Metadata from './metadata';
+
+
+export default class Nodule {
+
+    constructor(metadata, loaders = null, scope = null) {
```
Comment (pelegdvir) [inline src/nodule.js]: The default value for `scope` is actually 'nodule'.
So, the behavior of `Nodule(scope='nodule')` can be surprising for some.

**Pull request closed without merging**